Tests/Foundation/Tests/TestProcess.swift

```diff
@@ -699,6 +701,53 @@ class TestProcess : XCTestCase {
         }
     }
 
+    func test_currentDirectoryDoesNotChdirParentProcess() throws {
+        // This test only behaves correctly on Linux and Windows: other platforms don't have an
+        // appropriate API for this in posix_spawn or similar.
+        #if os(Linux) || os(Windows)
```

Comment:
Can this throw an XCTSkip instead? 

That way, maintainers of ports to other platforms (e.g. BSD) have a visible signal that this behaviour is not being verified, and can take any appropriate actions.

Reply:
Yes, it can.